### DISCUSSION TITLE
Jaeger Collector Istio Sidecar Exclusion

### DISCUSSION THREAD

**Discussion by timcosta** [jaegertracing/jaeger-operator#1659] [Q&A]
Hey all! I'm curious about the reasoning for jaeger-collector explicitly disabling the istio sidecar injection with no option to re-enable it. I check the code for comments, looked at the original PR, and search for issues but I don't see any discussion for why it was done. I'm trying to understand because it results in a lot of `unknown` destinations for workloads in our cluster that talk to it and have the sidecar enabled, and I can't come up with a reason why the injection would be hard disabled like it is. Anyone have any thoughts or know the reason?

**Comment by joshdcu** [comment]
Ditto. I went through the [`CHANGELOG.md`](https://github.com/jaegertracing/jaeger-operator/blob/main/CHANGELOG.md) and could only find two PRs (#67 and #1860) related to the Istio sidecar annotations and neither of them provide any insight into why it's being done. No dice with the [operator documentation](https://www.jaegertracing.io/docs/1.38/operator) either. Would really appreciate it if someone could explain this seemingly arbitrary decision! 😅

**Reply by joshdcu** [reply]
Almost been three weeks since I asked this 😅 @jpkrohling, I see that you approved #67. Could you please provide some insight into this decision? I apologise if there's a simple reason for this but I'm not able to understand it. One thing that I did read in the documentation is

> [Defining sampling strategies] is not relevant if a trace was started by the Istio proxy as the sampling decision is made there. And the Jaeger sampling decisions are only relevant when you are using the Jaeger tracer (client). [[reference](https://www.jaegertracing.io/docs/1.38/operator/#defining-sampling-strategies)]

But it's still not clear to me if this is the reason why it's done
